audresample/core/api.py

```diff
@@ -208,6 +194,10 @@ def resample(
     """
     signal = _check_signal(signal)
 
+    # We can only handle float32 signals
+    if signal.dtype != np.float32:
+        signal = signal.astype(np.float32)
```

Comment:
At the moment I silently convert to `float32` when resampling like we did before for `remix()` and `resample()`.
Should I replace this with raising an error instead?

Reply:
If yes, I would propose to do this in another pull request to have a commit for it.

Reply:
Yes, I think it's safer to raise an error otherwise we'll have a similar issue as reported in https://github.com/audeering/audinterface/issues/37 and users may wonder why it works with float64 when resampling is turned on.

Reply:
Created https://github.com/audeering/audresample/issues/18 to track this.